ui/src/Components/ProfilePage.jsx

```diff
@@ -36,7 +35,6 @@ export default class ProfilePage extends React.Component {
   render(){
     return(
       <div>
-        <NavBar />
```

Comment:
Why does the nav bar no longer persist in this page?

Reply:
Because I replaced it with one call in `App.js`  instead of a call in each view, it really cleaned things up!